crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -491,13 +491,16 @@ pub struct AuthorityEpochTables {
     pub(crate) randomness_next_round: DBMap<u64, RandomnessRound>,
     /// Holds the value of the highest completed RandomnessRound (as reported to RandomnessReporter).
     pub(crate) randomness_highest_completed_round: DBMap<u64, RandomnessRound>,
+    /// Holds the timestamp of the most recently generated round of randomness.
+    pub(crate) randomness_last_round_timestamp: DBMap<u64, TimestampMs>,
 }
 
 // TODO: move deferral related data structures to authority_per_epoch_store_util.rs
 #[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
 pub enum DeferralKey {
-    // For transactions deferred until randomness DKG has completed.
-    RandomnessDkg {
+    // For transactions deferred until new randomness is available (whether delayd due to
+    // DKG, or skipped commits).
+    Randomness {
         deferred_from_round: Round,
```

Comment:
this round now refers to a commit round (not randomness round), maybe change the name or clarify that in a comment

Reply:
added a comment, I think the type `Round` (same as below) and not `RandomnessRound` is another clue